data/sql/updates/pending_db_world/rev_1488880461400153400.sql

```diff
@@ -0,0 +1,21 @@
+INSERT INTO version_db_world (`sql_rev`) VALUES ('1488880461400153400');
```

Comment:
again not you fix, give credits to the author from https://github.com/dalaranwow/dalaran-wow/issues/4523